### ISSUE TITLE
Cannot read property 'addData' of undefined

### ISSUE THREAD

**Issue by nairtejas** [kshyju/VSTS-RecentCodeCommits#1]
Hi!

When I configure this widget in our team dashboard the widget fails to load and I get the following error:
"Cannot read property 'addData' of undefined"

I have left the "source code path" field empty and "Get data for" is set to "No filter". The only parameter supplied is the Widget title.

I'm using TFS 2017.
(Microsoft Visual Studio Team Foundation Server
Version 15.112.26307.0)

Regards,
Tejas Nair.

**Comment by kshyju** [comment]
Did you download the extension from the marketplace ?
[https://marketplace.visualstudio.com/items?itemName=kshyju.RecentCodeCommitsExt](https://marketplace.visualstudio.com/items?itemName=kshyju.RecentCodeCommitsExt)

**Comment by nairtejas** [comment]
Yes, that is what I did. I get this issue in only one of my team project. In the other project the widget seems to work fine. What could be the reason?

**Comment by kshyju** [comment]
That is strange. I will try to replicate it.
